org/wildfly/mod_cluster/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.mod_cluster
```

Comment:
https://github.com/wildfly/wildfly/pull/9803/files is what I thought this was about. It has requirements.